paddle/framework/operator.h

```diff
@@ -311,9 +332,9 @@ class InferShapeContext {
   }
 
   template <typename T>
-  std::vector<const T*> MultiOutput(const std::string& name) const {
+  std::vector<T*> MultiOutput(const std::string& name) const {
```

Comment:
I am not sure, should we name this DuplicableOutput?

Reply:
Yes, I think DuplicableOutput is more accerate